packages/components/src/tag/src/Tag.css

```diff
@@ -227,6 +227,7 @@ a.o-ui-tag-outline.o-ui-tag-disabled {
 
 /* CONTENT | REMOVE BUTTON */
 .o-ui-tag-remove-button {
+    border-radius: var(--hop-shape-rounded-sm);
```

Comment:
This is not any more specific than the other declaration, there could be issues if the closeButton css loads after.

Reply:
ok, fixed.